nfc/libnfc-nci-huawei.conf

```diff
@@ -0,0 +1,394 @@
+# This file from Honor View 10 stock pie, thanks @haky86
```

Comment:
Why/how are we assuming that Honor View 10 nfc conf file will fit every Huawei treble enabled phone?